frontend/src/pages/PredictedDataPage/PredictedDataPage.tsx

```diff
@@ -44,80 +57,122 @@ const PredictedDataPage = function () {
             newChecked.splice(currentIndex, 1);
             selectedFiles[value].isSelected = false;
         }
-
         setChecked(newChecked);
     };
 
-    const { uploadedFiles: fitBitFiles, error: predictedFitbitError } = useGetPredictedDataList(
-        WatchType.FITBIT,
-    );
-    if (predictedFitbitError) {
-        rollbar.error(predictedFitbitError.toString());
-    }
+    const getAvailableFiles = (): Array<PredictedFileWithType> => {
```

Comment:
i assume you just have this for testing?

Reply:
yepp I will use the useGetPredictedDataList method before merging